tests/unit/algorithms/segmentation/tasks/test_segmentation_inference.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import os
+
+import numpy as np
+import pytest
+
+from otx.algorithms.common.tasks import BaseTask
+from otx.algorithms.segmentation.tasks import SegmentationInferenceTask
+from otx.api.configuration.helper import create
+from otx.api.entities.annotation import Annotation
+from otx.api.entities.datasets import DatasetEntity
+from otx.api.entities.label import LabelEntity
+from otx.api.entities.metrics import Performance, ScoreMetric
+from otx.api.entities.model_template import parse_model_template
+from otx.api.entities.resultset import ResultSetEntity
+from otx.api.entities.scored_label import ScoredLabel
+from otx.api.entities.shapes.polygon import Point, Polygon
+from otx.api.usecases.evaluation.metrics_helper import MetricsHelper
+from otx.api.usecases.tasks.interfaces.export_interface import ExportType
+from otx.api.utils.shape_factory import ShapeFactory
+from tests.test_suite.e2e_test_system import e2e_pytest_unit
+from tests.unit.algorithms.segmentation.test_helpers import (
+    DEFAULT_SEG_TEMPLATE_DIR,
+    create_model,
+    generate_otx_dataset,
+    init_environment,
+)
+
+
+class TestOTXSegTaskInference:
+    @pytest.fixture(autouse=True)
+    def setup(self) -> None:
+        model_template = parse_model_template(os.path.join(DEFAULT_SEG_TEMPLATE_DIR, "template.yaml"))
+        hyper_parameters = create(model_template.hyper_parameters.data)
+        task_env = init_environment(hyper_parameters, model_template)
+        self.seg_train_task = SegmentationInferenceTask(task_env)
```

Comment:
I think this code makes something in `/tmp` directory and not deletes it because you don't set `output_path` when initializing `SegmentationInferenceTask`. How about making temporary directory using [tempfile](https://docs.python.org/3/library/tempfile.html) and set it as `output_path` argument to delete it automatically after test?

Reply:
I revised based on your review. Could you check again?